src/main/java/bio/terra/workspace/service/datareference/model/CloningInstructions.java

```diff
@@ -33,4 +42,9 @@ public static CloningInstructions fromApiModel(CloningInstructionsEnum modelEnum
   public CloningInstructionsEnum toApiModel() {
     return instructionMap.get(this);
   }
+
+  /** Convert this to a String to be serialized to the DB. */
+  public String toSql() {
+    return sqlMap.get(this);
+  }
```

Comment:
kick me for scope creep ... now I'm wondering if we need a corresponding `fromSql` to use instead of relying on `valueOf` ... even if the initial implementation of `fromSql` delegates to `valueOf`.

Reply:
Good point - for the sake of consistency that's the right thing to do. I think this is another good place for a `BiMap`